spec.html

```diff
@@ -177,7 +177,6 @@ <h1>get _AggregateError_.prototype.errors</h1>
         <emu-alg>
           1. Let _E_ be the *this* value.
           1. If Type(_E_) is not Object, throw a *TypeError* exception.
-          1. If _E_ does not have an [[ErrorData]] internal slot, throw a *TypeError* exception.
```

Comment:
This isn’t redundant, since the intention is to ensure that both constructors have run on the object. Using prototype manipulation, you could have an object that has the AggregateError slot but not the Error slot.

Reply:
How exactly?

Reply:
I ask because https://tc39.es/proposal-promise-any/#sec-aggregate-error step 2 says:

```
Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]], [[AggregateErrors]] »).
```

...thus ensuring both are present.

Reply:
That’s when using Promise.any, but not when using new AggregateError.

Reply:
Hmm, i see what you mean. In the current spec text, this line is implicitly redundant - but editorially i prefer the explicit double check to prevent future spec authoring errors.

Reply:
We could change https://tc39.es/proposal-promise-any/#sec-properties-of-aggregate-error-instances from:

> AggregateError instances are ordinary objects that inherit properties from their AggregateError prototype object and have an [[ErrorData]] internal slot whose value is undefined.

to something like:

> AggregateError instances are ordinary objects that inherit properties from their AggregateError prototype object and have an [[ErrorData]] internal slot whose value is undefined, as well as an [[AggregateErrors]] internal slot whose value is undefined.

Would that be clearer?

Reply:
Only if the line below this was also changed to assert that it’s an aggregate error instance rather than checking for slots.

Reply:
Ok, let's do that then. @marjakh, do you want to update the patch accordingly?

Reply:
The patch has been updated. @ljharb, do you want to take another look?